assets/src/edit-story/panels/panel/content.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import panelContext from './context';
+
+const Form = styled.form`
+	margin: 10px 20px;
+	overflow: auto;
+`;
+
+function Content( { children, ...rest } ) {
+	const { state: { isCollapsed, height } } = useContext( panelContext );
+
+	if ( isCollapsed ) {
+		return null;
```

Comment:
I was wonder, if this doesn't render, does it have accessibility impacts here?

Reply:
This shouldn't be a problem because users would never be able to tab into a non-expanded area.

Gutenberg also doesn't have any special handling in this regard.

Reply:
Hadn't thought about this. Will leave the comment open for discussion. I haven't seen any recommendations in this regard.

Reply:
The W3 example add `hidden` attribute and hides with CSS. Replicated here now.